.github/actions/prepare-deps-win32/action.yml

```diff
@@ -19,10 +19,18 @@ runs:
         choco install `
           jom `
           nasm `
-          nodejs
```

Comment:
I suspect removing nodejs will make building the web UI impossible. Not sure if it's critical but also not sure why this change is necessary.

Reply:
github windows-11-arm runner use chocolate install nodejs will cause an error, i am not sure why that is. and all runner is preinstalled nodejs. so remove it.

Reply:
https://github.com/actions/runner-images/blob/main/images/windows/Windows2022-Readme.md

Reply:
Runner pre-installed Node.js , so we don't need to install it again.